components/camera/server.go

```diff
@@ -53,8 +53,9 @@ func (s *subtypeServer) GetImage(
 		return nil, err
 	}
 
-	if req.MimeType == "" {
-		req.MimeType = utils.MimeTypeRawRGBALazy
+	err = utils.CheckSupportedMimeType(req.MimeType)
```

Comment:
I struggled to find a suitable default because I didn't want to default to something that used our custom header, but bringing back the lazy mime type should address that